modules/programs/zsh.nix

```diff
@@ -547,10 +548,12 @@ in
 
         # Aliases
         ${aliasesStr}
-
-        # Global Aliases
-        ${globalAliasesStr}
```

Comment:
Can we remove the definition of `globalAliasesStr` from above in the file?

Reply:
good catch